src/main/java/network/brightspots/rcv/TieBreak.java

```diff
@@ -118,15 +118,15 @@ String nonSelectedCandidateDescription() {
       nonselected = options.get(0);
     } else if (options.size() == 2) {
       // if there are only 2 candidates don't use a comma
-      nonselected = options.get(0) + " and " + options.get(1);
+      nonselected = '"' + options.get(0) + "\" and \"" + options.get(1) + '"';
```

Comment:
It's a little weird to inconsistently render quotes like this in our messages. I recommend consistently sticking with `\"` instead of `'"'`, since that's what I've been doing elsewhere. If you change here, please change elsewhere too.